arrows/vxl/average_frames.h

```diff
@@ -0,0 +1,53 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#ifndef KWIVER_ARROWS_VXL_AVERAGE_FRAMES_
+#define KWIVER_ARROWS_VXL_AVERAGE_FRAMES_
+
+#include <arrows/vxl/kwiver_algo_vxl_export.h>
+
+#include <vital/algo/image_filter.h>
+
+namespace kwiver {
+namespace arrows {
+namespace vxl {
+
+/**
+ * @brief VXL Frame Averaging Process
+ *
+ * This method contains basic methods for image filtering on top of input
+ * images via performing assorted averaging operations.
+ */
+class KWIVER_ALGO_VXL_EXPORT average_frames
+  : public vital::algorithm_impl< average_frames,
```

Comment:
Is `algorithm_impl` needed any more as a base class?

Reply:
No, and there's actually a deprecation warning. I just hadn't addressed it previously, thought it's done now.